features/daily/line.feature

```diff
@@ -14,3 +14,13 @@ Feature: Line
     Then I have the following hints:
       | exten        | line              |
       | 1603@default | PJSIP/newusername |
+
+  Scenario: Custom line
+    Given there are telephony users with infos:
+      | firstname | lastname | exten | context | protocol | with_phone |
+      | User      | SIP      | 1801  | default | sip      | yes        |
+      | User      | Custom   | 1802  | default | custom   | no        |
+    Given "User Custom" has custom interface equal to SIP interface from "User SIP"
```

Comment:
The interface could have been `Local/1801@default` to make the step implementation simpler

Reply:
weeeeee :tada: